package.nls.json

```diff
@@ -910,6 +910,7 @@
     },
     "DataScience.pythonExtensionRequired": "The Python Extension is required to perform that task. Click Yes to open Python Extension installation page.",
     "DataScience.pythonExtensionRequiredToRunNotebook": "Python Extension required to run Python notebooks.",
+    "DataScience.installingPythonExtension": "Installing Python extension and locating Python kernels.",
```

Comment:
I'd consider changing this to `location Kernels`. as we can end up location non-python kernels that are installed in conda environments.